bitbucket/resource_branching_model.go

```diff
@@ -181,7 +184,7 @@ func resourceBranchingModelsRead(ctx context.Context, d *schema.ResourceData, m
 	if err != nil {
 		return diag.FromErr(err)
 	}
-	branchingModelsReq, _ := client.Get(fmt.Sprintf("2.0/repositories/%s/%s/branching-model", owner, repo))
+	branchingModelsReq, _ := client.Get(fmt.Sprintf("2.0/repositories/%s/%s/branching-model/settings", owner, repo))
```

Comment:
The old code used the wrong API. The two APIs are similar, but the old one omits disabled branches in `branch_types`, and it omits `production` if disabled. Using `settings` returns everything. Note that the code was already using `settings` for `PUT` operations. This change aligns `GET` and `PUT` to use the same URL.